server/db_data_init.py

```diff
@@ -0,0 +1,198 @@
+        "status_id": 1,
+        "role_id": 1
+    },
+    {
+        "email": "geraltemail@gmail.com",
+        "nickname": "WhiteWolf",
+        "password": pbkdf2_sha256.hash("Merigold"),
+        "location": "Kaer Morhen",
+        "first_name": "Geralt",
+        "last_name": "of Rivia",
+        "status_id": 4,
+        "role_id": 2
+    }
+)
+event_data = (
+    {
+        "name": "Oleh Vynnyk Rivne Show",
+        "long": 26.251617,
+        "lat": 50.619900,
+        "description": "Some show blah",
+        "start_date": "2017-12-21",
+        "end_date": "2017-12-21",
+        "author_name": "what",
+        "main_image": "vynnykposter",
+        "author_id": 4,
+        "category_id": 2
+    },
+    {
+        "name": "Dark Elf Adventure",
+        "long": 27.251617,
+        "lat": 51.619900,
+        "description": "Drizzt long adventure",
+        "start_date": "2019-11-01",
+        "end_date": "2019-11-05",
+        "author_name": "Drizzt",
+        "main_image": "drizztcoolposter",
+        "author_id": 3,
+        "category_id": 3
+    },
+    {
+        "name": "Independent day",
+        "long": 38.907192,
+        "lat": -77.036871,
+        "description": "Beer, burgers, rock n roll",
+        "start_date": "2018-07-04",
+        "end_date": "2018-07-05",
+        "author_name": "Oleksandr",
+        "main_image": "someposter",
+        "author_id": 1,
+        "category_id": 1
+    }
+)
+event_tag_data = (
+    {"event_id": 1, "tag_id": 1},
+    {"event_id": 1, "tag_id": 2},
+    {"event_id": 1, "tag_id": 7},
+    {"event_id": 2, "tag_id": 3},
+    {"event_id": 2, "tag_id": 5},
+    {"event_id": 3, "tag_id": 4},
+    {"event_id": 2, "tag_id": 5}
+)
+event_history_data = (
+    {"event_id": 1, "status_id": 2,
+     "date": "2017-12-21", "comment": "omg vynnyk best"},
+    {"event_id": 3, "status_id": 2,
+     "date": "2018-07-05", "comment": "wow was fun"},
+    {"event_id": 2, "status_id": 3, 
+     "date": "2019-11-01", "comment": "coming soon"}
+)
+gallery_data = (
+    {"img_url": "somevynnykurl", "event_id": 1},
+    {"img_url": "somedrizzturl", "event_id": 2},
+    {"img_url": "someindependenturl", "event_id": 3}
+)
+feedback_data = (
+    {
+        "user_id": 2,
+        "event_id": 3,
+        "feedback": "where was i",
+        "date": "2018-07-06",
+        "is_deleted": False
+    },
+    {
+        "user_id": 4,
+        "event_id": 1,
+        "feedback": "VYNNYK NUMBER 1!",
+        "date": "2017-12-21",
+        "is_deleted": True
+    },
+    {
+        "user_id": 1,
+        "event_id": 2,
+        "feedback": "Drizzt or better Dzirt",
+        "date": "2018-07-01",
+        "is_deleted": False}
+    )
+subscribe_data = (
+    {"user_id": 4, "event_id": 1, "is_favorite": False},
+    {"user_id": 2, "event_id": 2, "is_favorite": True},
+    {"user_id": 5, "event_id": 2, "is_favorite": True}
+)
+
+conn = engine.connect()
+conn.execute(role.Role.__table__.insert(), role_data)
```

Comment:
In my opinion, the loop 'for' will look better:
list_1 = [role.Role, tag.Tag, category.Category, user_status.UserStatus, event_status.EventStatus,
          user.User, event.Event, event_tag.EventTag, event_history.EventHistory, gallery.Gallery,
          feedback.Feedback, subscribe.Subscribe]

list_2 = [role_data, tag_data, category_data, user_status_data, event_status_data, user_data,
          event_data, event_tag_data, event_history_data, gallery_data, feedback_data, subscribe_data]

conn = engine.connect()
for i in range(0, len(list_1)):
    conn.execute(list_1[i].__table__.insert(), list_2[i])

Reply:
I have one more idea.

Reply:
I'm not sure it's a good idea, i'd rather keep a testing data separated in this module.